package.json

```diff
@@ -9,6 +9,7 @@
     "lint": "run-s -c lint:*",
     "lint:prettier": "prettier --check \"src/**/**/!(*.d).{ts,json}\"",
     "lint:eslint": "eslint \"src/**/*.ts\"",
+    "prettier": "prettier --write \"**/**/!(*.d).{ts,json}\"",
```

Comment:
Calling this `prettier` prevents you from calling prettier directly from yarn, maybe `format` or `fix`.
Also, we should probably match the glob to the one in `lint:prettier`

Reply:
Fair point - fixed in https://github.com/EventStore/EventStore-Client-NodeJS/pull/90/commits/09999d7f54bca348553b5269d9ed417609703ba3#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R12.